src/jdk.crypto.mscapi/windows/native/libsunmscapi/security.cpp

```diff
@@ -415,8 +416,22 @@ JNIEXPORT void JNICALL Java_sun_security_mscapi_CKeyStore_loadKeysOrCertificateC
             == NULL) {
             __leave;
         }
-        if ((hCertStore = ::CertOpenSystemStore(NULL, pszCertStoreName))
+        if ((pszCertStoreLocation = env->GetStringUTFChars(jCertStoreLocation, NULL))
```

Comment:
Would it be easier to use a int argument here, it’s a internal api with only two enumerated Types. That safes the jstring conversion and the Strcmp.

Reply:
And also, is there a ReleaseString missing?

Reply:
Thanks for the feedback, I'm going to incorporate that into the PR

Reply:
> And also, is there a ReleaseString missing?

Yes an error when I "patched" my repo, but based on the feedback there will no longer be a string to free :)

Reply:
Committed changes to use ints instead of strings